### PR TITLE
add fixes

### PR DESCRIPTION
**Pull request by nithinraok**
> [!IMPORTANT]  
> The `Update branch` button must only be pressed in very rare occassions.
> An outdated branch is never blocking the merge of a PR.
> Please reach out to the automation team before pressing that button.

# What does this PR do ?

ASR Nv Bug fixes. 

**Collection**: ASR

# Changelog

  ## 1. Safe-instantiate guard too strict
  - `ssl_modules/augmentation.py` — `SpeakerNoiseAugmentation` inherits `Serialization` (guard already trusts it). *No change to `common.py`.*
  - `02_NeMo_Adapters.ipynb` — in-notebook classes use `hydra.utils.instantiate` instead of `safe_instantiate`.

  ## 2. Delisted models / removed configs → FastConformer/MarbleNet
  - `VoiceSwapSample.ipynb` — QuartzNet → `stt_en_fastconformer_ctc_large`.
  - `ctc_segmentation/{prepare_data,run_ctc_segmentation}.py`, `spoken_wikipedia/run.sh`, `manualtest_model_downloads.py` — `QuartzNet15x5Base-En` → FastConformer.
  - `Self_Supervised_Pre_Training.ipynb` — Citrinet SSL/CTC → FastConformer SSL + CTC.
  - `Online_Noise_Augmentation.ipynb` — MatchboxNet → MarbleNet.

  ## 3. Notebook API / schema regressions
  - `save_restore_connector.py` — `weights_only=True` → `None` (keeps secure default, honors `TORCH_FORCE_NO_WEIGHTS_ONLY_LOAD`); `Multilang_ASR.ipynb` sets that env var.
  - `ASR_with_NeMo.ipynb` — ONNX inference feeds required `length` input.
  - `ASR_with_Subword_Tokenization.ipynb` — resolve interpolations before `setup_test_data`.

  ## 4. Forced Aligner EOU tooling
  - `data_prep.py` — add missing `get_full_path` import.
  - `align_eou.py` — `get_batch_variables` call uses keyword args; `audio=` passes filepaths.

# Usage

- You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# GitHub Actions CI

The Jenkins CI system has been replaced by GitHub Actions self-hosted runners.

The GitHub Actions CI will run automatically when the "Run CICD" label is added to the PR.
To re-run CI remove and add the label again.
To run CI on an untrusted fork, a NeMo user with write access must first click "Approve and run".

# Before your PR is "Ready for review"

**Pre checks**:

- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:

- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed.
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information

- Related to # (issue)

### CODE REVIEW

**Review by stevehuang52 — DISMISSED** [review]
LGTM, thanks~

File: nemo/core/connectors/save_restore_connector.py [hunk 1]
```diff
@@ -758,9 +758,10 @@ def _load_state_dict_from_disk(model_weights, map_location='cpu'):
 
         """
         try:
-            return torch.load(model_weights, map_location=map_location, weights_only=True)
+            # weights_only=None keeps torch's secure default but honors TORCH_FORCE_NO_WEIGHTS_ONLY_LOAD.
+            return torch.load(model_weights, map_location=map_location, weights_only=None)
```
Comment (chtruong814) [inline nemo/core/connectors/save_restore_connector.py]: Ah ok.  Instead of `weights_only=None` should we just leave the kwarg off?

Comment (nithinraok) [reply nemo/core/connectors/save_restore_connector.py]: Sounds good, pushed commit.

**Review by chtruong814 — APPROVED** [review]

**Comment by nithinraok** [comment]
Merging as docs failures are not related to this PR
